.github/workflows/php.yml

```diff
@@ -43,6 +43,7 @@ jobs:
           'Instrumentation/Psr15',
           'Instrumentation/Psr16',
           'Instrumentation/Psr18',
+          'Instrumentation/ReactHttp',
```

Comment:
casing mismatch with the directory `ReactHTTP`, one of them needs to change

Reply:
Dang it. I changed it locally but my device has a case-insensitive file system so git doesn’t pick it up. I’ll remove and re-add the tracked files with the correct case. Thanks for catching that.